README.md

```diff
@@ -141,12 +141,37 @@ build_windows_cuda.bat
 Build can take ~2 hours depending on the machine configuration.
 
 ### Build and test using docker
+
+### Prerequisites
+* Ensure virtualization is enabled in both the BIOS settings of your computer and the Windows virtualization feature
```

Comment:
Can we link some article here ? I know the BIOS may be different on may computers but the second half "windows vistualization feature" it is not clear how to do that ?

Reply:
will look for an article which has some general way and i will link that here

Reply:
linked